src/gui/menu/description.py

```diff
@@ -57,7 +67,8 @@ def mouse_wheel(self, event) -> bool:
 
     # draw
     def make_surface_transparent(self):
-        self.description_surface.fill(pygame.Color(0, 0, 0, 0))
+        self.description_surface.fill('green')
+        self.description_slider_surface.fill('green')
```

Comment:
Same comment

Reply:
As for all further instances of this, you need to fill the images with the transparent colour first. If we were to decide using green for the sliders, we would get an unexpected rendering fault due to the colourkey existing, which is why I prefer using alpha channels instead

```suggestion
        self.description_surface.fill(pygame.Color(0, 0, 0, 0))
        self.description_slider_surface.fill(pygame.Color(0, 0, 0, 0))
```